src/posts/paper/14-research-sources.ts

```diff
@@ -54,6 +52,10 @@ We combine these numbers to estimate approximately a 4x reduction. Again, this i
 
 We note that we think masks on the infected person might not help proportionately as much at larger indoor distances, because surgical masks are most effective at stopping larger droplets ([Leung et al.](https://www.nature.com/articles/s41591-020-0843-2)).
 
+For n95 masks, we get our figure from [Jimenez et al.](https://docs.google.com/spreadsheets/d/16K1OQkLD4BjgBdO8ePj6ytf-RpPMlJ6aXFg3PrIQBbQ/edit#gid=519189277&range=C190:J190) who suggest that, while a perfect fit with an n95 is 99% efficient at filtering out aerosols, average use causes them to discount this to 90%.
```

Comment:
N95 has a capital N.

Referencing Jimenez is a little awkward because we disagree with his other mask figures that are right next to that one. Chu might be a better source, or maybe there's something else we can dig up?

Reply:
Chu says that N95 was about 10x better than other masks (0.04 vs 0.33), so we'd need to make the N95 multiplier 20x instead of 10x. Do you want me to do that? Or say we are discounting to 10x to account for fit?

Reply:
Done.